### ISSUE TITLE
Banners do not pass AA, throws contrast errors 

### ISSUE THREAD

**Issue by olitharp-nci** [NCIOCPL/ncids#347]
### Issue description

The banner component throws errors about contrast. The NCI component is not using this color, but the default USWDS component does use it.

> **ESTIMATE** TBD

![Screen Shot 2022-02-25 at 10 58 38 AM](https://user-images.githubusercontent.com/85901679/155756877-6508e336-59cc-40dd-ae7d-9fe734cf4692.png)
![Screen Shot 2022-02-25 at 10 59 25 AM](https://user-images.githubusercontent.com/85901679/155756891-87e954b7-b939-4651-ba49-270078bc40a8.png)

```bash
Warning: [Banner] Neither the specified preferred link color token (`base-lighter`) nor the fallback link color token (`primary`) have AA contrast on a `base-lightest` background. Using `primary`. Please check your source code and project settings.
    ../node_modules/uswds/dist/scss/core/_functions.scss 1936:7               get-color-token-from-bg()
    ../node_modules/uswds/dist/scss/core/_functions.scss 2010:27              get-link-tokens-from-bg()
    ../node_modules/uswds/dist/scss/core/mixins/_set-link-from-bg.scss 10:17  set-link-from-bg()
    ../node_modules/uswds/dist/scss/components/_banner.scss 231:5             @content
    ../node_modules/uswds/dist/scss/core/mixins/_at-media.scss 15:7           at-media()
    ../node_modules/uswds/dist/scss/components/_banner.scss 223:3             @import
    ../packages/ncids-css/scss/components/usa-banner/_nci-banner.scss 5:9     @import
    src/scss/_components.scss 28:9                                            @import
    src/index.scss 22:9    
```

### Related Tickets
* Issue #495

**Comment by bryanpizzillo** [comment]
This should be resolved by #352

**Comment by blilianyu** [comment]
Can we change the link color in the USA banner to primary-dark (#004971) to resolve the 508 issue?

**Comment by bennettcc** [comment]
Per 9/19 meeting - this story should be rethought. Colors/styles that are not accessible are being suggested.

If prioritized, we'll need to regroup on this @blilianyu @laurelthrash (perhaps in a future design check in)

**Comment by sarinapadilla** [comment]
Per 11/10 dev backlog meeting: we need to upgrade to USWDS 3.0. During that upgrade, we will need to figure out how to stop these errors showing up.

**Comment by bryanpizzillo** [comment]
This should be fixed by #978. However I have used this phrase before and was wrong.

**Comment by bennettcc** [comment]
This will be resolved by #978

**Comment by bryanpizzillo** [comment]
Actually closing this as duplicated by #978
